### PR TITLE
PLIO to ISIS pixel type conversion 

### PR DESCRIPTION
**Pull request by Kelvinrr**

### CODE REVIEW

File: autocnet/spatial/isis.py [hunk 1]
```diff
@@ -33,11 +33,26 @@ def point_info(cube_path, x, y, point_type):
...
+
+
     if isinstance(x, Number) and isinstance(y, Number):
         x, y = [x], [y]
 
     with tempfile.NamedTemporaryFile("w+") as f:
         # ISIS wants points in a file, so write to a temp file
+        if point_type == "ground":
+            # for reasons beyond comprehension, campt using lat, lon for ground
+            # but sample, line for image.
```
Comment (jessemapel) [inline autocnet/spatial/isis.py]: ISIS is our software, we probably shouldn't be harassing ourselves like this.

Comment (Kelvinrr) [reply autocnet/spatial/isis.py]: :+1:

**Comment by jlaura** [comment]
Does this need a rebase? I thought some of these commits were merged earlier today?

File: autocnet/spatial/isis.py [hunk 2]
```diff
@@ -33,11 +33,26 @@ def point_info(cube_path, x, y, point_type):
     : PvlObject
       Pvl object containing campt returns
     """
+    point_type = point_type.lower()
+
+    if point_type not in {"image", "ground"}:
+        raise Exception(f'{point_type} is not a valid point type, valid types are ["image", "ground"]')
+
```
Comment (jlaura) [inline autocnet/spatial/isis.py]: :+1: I am digging this syntax - very nice not to have to read to the end of the line for the .format and then substitute.

**Comment by Kelvinrr** [comment]
yeah, just did a pull from the same branch and re-PRed

**Comment by Kelvinrr** [comment]
should be good to go after Travis is done.

**Comment by jlaura** [comment]
:+1: Feel free to merge when CI passes.

**Pull request merged**